app/controllers/api/v1/moves_controller.rb

```diff
@@ -7,22 +7,29 @@ def index
         moves_params = Moves::ParamsValidator.new(filter_params, params[:sort] || {})
         if moves_params.valid?
           moves = Moves::Finder.new(filter_params, current_ability, params[:sort] || {}).call
-          paginate moves, include: MoveSerializer::INCLUDED_ATTRIBUTES
+          paginate moves, include: MoveSerializer::INCLUDED_ATTRIBUTES.dup.except(:court_hearings)
```

Comment:
it would be good to add a short comment to explain why court hearings are intentionally excluded in GET moves but included in a single GET move - otherwise we might think its a bug later down the line